ide/coqide/coq.ml

```diff
@@ -611,7 +612,14 @@ let db_configd x = eval_call ~db:true (Xmlprotocol.db_configd x)
 
 let interrupt_coqtop coqtop workers =
   if coqtop.status = Busy then
-    try !interrupter (CoqTop.unixpid coqtop.handle.proc)
+    try
+      let pid = CoqTop.unixpid coqtop.handle.proc in
```

Comment:
Why is this OS-specific code specifically there, when we already have a dedicated file to set the `interrupter` value? Can't you move it to `ide/coqide/coqide_WIN32.c.in`?

Reply:
Got it--didn't occur to me.